azure/iac/sql_servers.rego

```diff
@@ -16,10 +23,19 @@ azure_issue["sql_public_access"] {
 
 sql_public_access {
     lower(input.resources[_].type) == "microsoft.sql/servers"
+    not azure_attribute_absence["sql_public_access"]
     not azure_issue["sql_public_access"]
 }
 
 sql_public_access = false {
+    azure_attribute_absence["sql_public_access"]
+}
+
+sql_public_access = false {
+    azure_issue["sql_public_access"]
+}
+
+sql_public_access_miss_err = "SQL servers attribute public access missing in the resource!" {
     azure_issue["sql_public_access"]
```

Comment:
this line should be `azure_attribute_absence["sql_public_access"]`

Reply:
fixed